PEP8 in top-level tests

dipy/tests/scriptrunner.py:21:5: E261 at least two spaces before inline comment
dipy/tests/scriptrunner.py:23:18: E261 at least two spaces before inline comment
dipy/tests/scriptrunner.py:30:39: E714 test for object identity should be 'is not'
dipy/tests/scriptrunner.py:36:11: E225 missing whitespace around operator
dipy/tests/scriptrunner.py:70:29: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:70:31: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:71:29: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:71:31: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:72:33: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:72:35: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:73:34: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:73:36: E251 unexpected spaces around keyword / parameter equals
dipy/tests/scriptrunner.py:73:45: E203 whitespace before ':'
dipy/tests/scriptrunner.py:74:17: E124 closing bracket does not match visual indentation
dipy/tests/scriptrunner.py:123:12: E714 test for object identity should be 'is not'
dipy/tests/scriptrunner.py:124:80: E501 line too long (83 > 79 characters)
dipy/tests/scriptrunner.py:125:80: E501 line too long (82 > 79 characters)
dipy/tests/scriptrunner.py:126:80: E501 line too long (82 > 79 characters)
dipy/tests/scriptrunner.py:127:80: E501 line too long (82 > 79 characters)
dipy/tests/scriptrunner.py:142:12: E714 test for object identity should be 'is not'
dipy/tests/scriptrunner.py:143:80: E501 line too long (83 > 79 characters)
dipy/tests/test_scripts.py:22:49: W291 trailing whitespace
dipy/tests/test_scripts.py:32:16: E251 unexpected spaces around keyword / parameter equals
dipy/tests/test_scripts.py:32:18: E251 unexpected spaces around keyword / parameter equals
dipy/tests/test_scripts.py:33:20: E251 unexpected spaces around keyword / parameter equals
dipy/tests/test_scripts.py:33:22: E251 unexpected spaces around keyword / parameter equals
dipy/tests/test_scripts.py:38:1: E302 expected 2 blank lines, found 1
dipy/tests/test_scripts.py:96:80: E501 line too long (84 > 79 characters)
dipy/tests/test_scripts.py:116:1: E302 expected 2 blank lines, found 1

--- a/dipy/tests/scriptrunner.py
+++ b/dipy/tests/scriptrunner.py
@@ -18,22 +18,22 @@ from os.path import (dirname, join as pjoin, isfile, isdir, realpath, pathsep)
 
 from subprocess import Popen, PIPE
 
-try: # Python 2
+try:  # Python 2
     string_types = basestring,
-except NameError: # Python 3
+except NameError:  # Python 3
     string_types = str,
 
 
 def _get_package():
     """ Workaround for missing ``__package__`` in Python 3.2
     """
-    if '__package__' in globals() and not __package__ is None:
+    if(('__package__' in globals()) and (__package__ is not None)):
         return __package__
     return __name__.split('.', 1)[0]
 
 
 # Same as __package__ for Python 2.6, 2.7 and >= 3.3
-MY_PACKAGE=_get_package()
+MY_PACKAGE = _get_package()
 
 
 def local_script_dir(script_sdir):
@@ -66,12 +66,12 @@ class ScriptRunner(object):
     Finds local scripts and local modules if running in the development
     directory, otherwise finds system scripts and modules.
     """
+
     def __init__(self,
-                 script_sdir = 'scripts',
-                 module_sdir = MY_PACKAGE,
-                 debug_print_var = None,
-                 output_processor = lambda x : x
-                ):
+                 script_sdir='scripts',
+                 module_sdir=MY_PACKAGE,
+                 debug_print_var=None,
+                 output_processor=lambda x: x):
         """ Init ScriptRunner instance
 
         Parameters
@@ -120,12 +120,13 @@ class ScriptRunner(object):
             cmd = [cmd]
         else:
             cmd = list(cmd)
-        if not self.local_script_dir is None:
-            # Windows can't run script files without extensions natively so we need
-            # to run local scripts (no extensions) via the Python interpreter.  On
-            # Unix, we might have the wrong incantation for the Python interpreter
-            # in the hash bang first line in the source file.  So, either way, run
-            # the script through the Python interpreter
+        if self.local_script_dir is not None:
+            # Windows can't run script files without extensions
+            # natively so we need to run local scripts (no extensions)
+            # via the Python interpreter.  On Unix, we might have the
+            # wrong incantation for the Python interpreter
+            # in the hash bang first line in the source file. So, either way,
+            # run the script through the Python interpreter
             cmd = [sys.executable,
                    pjoin(self.local_script_dir, cmd[0])] + cmd[1:]
         elif os.name == 'nt':
@@ -139,10 +140,10 @@ class ScriptRunner(object):
         if self.debug_print:
             print("Running command '%s'" % cmd)
         env = os.environ
-        if not self.local_module_dir is None:
-            # module likely comes from the current working directory. We might need
-            # that directory on the path if we're running the scripts from a
-            # temporary directory
+        if self.local_module_dir is not None:
+            # module likely comes from the current working directory.
+            # We might need that directory on the path if we're running
+            # the scripts from a temporary directory
             env = env.copy()
             pypath = env.get('PYTHONPATH', None)
             if pypath is None:
@@ -151,7 +152,7 @@ class ScriptRunner(object):
                 env['PYTHONPATH'] = self.local_module_dir + pathsep + pypath
         proc = Popen(cmd, stdout=PIPE, stderr=PIPE, env=env)
         stdout, stderr = proc.communicate()
-        if proc.poll() == None:
+        if proc.poll() is None:
             proc.terminate()
         if check_code and proc.returncode != 0:
             raise RuntimeError(

--- a/dipy/tests/test_scripts.py
+++ b/dipy/tests/test_scripts.py
@@ -19,7 +19,7 @@ from nibabel.tmpdirs import InTemporaryDirectory
 
 from dipy.data import get_data
 
-# Quickbundles command-line requires matplotlib: 
+# Quickbundles command-line requires matplotlib:
 try:
     import matplotlib
     no_mpl = False
@@ -29,11 +29,12 @@ except ImportError:
 from .scriptrunner import ScriptRunner
 
 runner = ScriptRunner(
-    script_sdir = 'bin',
-    debug_print_var = 'NIPY_DEBUG_PRINT')
+    script_sdir='bin',
+    debug_print_var='NIPY_DEBUG_PRINT')
 run_command = runner.run_command
 
 DATA_PATH = abspath(pjoin(dirname(__file__), 'data'))
+
 
 def test_dipy_peak_extraction():
     # test dipy_peak_extraction script
@@ -93,7 +94,8 @@ def test_dipy_fit_tensor_again():
         shutil.copyfile(bval, "small_25.bval")
         shutil.copyfile(bvec, "small_25.bvec")
         # Call script
-        cmd = ["dipy_fit_tensor", "--save-tensor", "--mask=none", "small_25.nii.gz"]
+        cmd = ["dipy_fit_tensor", "--save-tensor",
+               "--mask=none", "small_25.nii.gz"]
         out = run_command(cmd)
         assert_equal(out[0], 0)
         # Get expected values
@@ -112,6 +114,7 @@ def test_dipy_fit_tensor_again():
         ten_shape = shape + (1, 6)
         assert_image_shape_affine("small_25_tensor.nii.gz", ten_shape,
                                   affine)
+
 
 @nt.dec.skipif(no_mpl)
 def test_qb_commandline():

